webrender_traits/src/channel_mpsc.rs

```diff
@@ -3,7 +3,6 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 use serde::{Deserialize, Serialize};
-use std::io::Error;
```

Comment:
This is my version of the same change: https://github.com/jrmuizel/webrender/commit/cdcf5af0a8d336563ca652b5e2aec5426001d2de